client/src/components/LeafletOsmMap/SavedPlaces.js

```diff
@@ -0,0 +1,159 @@
+import {h, Component} from "preact";
+import {makeRequest} from "../../js/server-requests-utils";
+
+/**
+ * Leaflet related imports: leaflet, pouchdb module, and routing machine module
+ */
+import '../../../node_modules/leaflet/dist/leaflet.css';
+import '../../../node_modules/leaflet-routing-machine/dist/leaflet-routing-machine.css';
+import L from '../../js/leaflet-tileLayer-pouchdb-cached';
```

Comment:
still thinking about how best to handle this but we should probably pass the L object into you're `fetchAndDropUserPins` function. Actually, rather, we should pass the `FAV_MARKER_ICON` to it instead and move `FAV_MARKER_ICON` into `LeafletOsmMap/index.js`

Reply:
I refactored so that `FAV_MARKER_OPTIONS` are defined in the now `saved-places.js` but favorite markers are only rendered if `L` is passed in as an argument, otherwise it displays default icons